.github/renovate-base.json5

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "github>newrelic/coreint-automation:renovate-base.json5"
+  ],
+  ignoreDeps: ["ingress-nginx/kube-webhook-certgen"],
+  "customManagers": [
```

Comment:
We need to group all dependencies related to Minikube, like it was done here:
https://github.com/newrelic/nri-kubernetes/blob/main/.github/renovate.json5#L11-L14

Reply:
The grouping looks like this:
```
  "packageRules": [
    {
      "groupName": "Minikube version (batch update)",
      "matchDepNames": ["manusa/actions-setup-minikube", "kubernetes/minikube"],
    },
  ],
```

Reply:
Done